src/exporter.py

```diff
@@ -134,7 +136,7 @@ def install(self, snap_path: Optional[str] = None) -> None:
             snap.snap_install(snap_path, "--dangerous")
         else:
             logger.info("Installing %s snap from snap store.", self.SNAP_NAME)
-            snap.snap_install(self.SNAP_NAME)
+            snap.snap_install(self.SNAP_NAME, "--channel", snap_channel)
```

Comment:
given that this is an ops charm we kind of should be using the snap ops library: https://charmhub.io/operator-libs-linux/libraries/snap

Reply:
This is a bit out-of-scope for this PR. I think it should be addressed as part of #18 (Replacing charmhelpers).